src/components/specific/tags/tags-main/TagsMain.vue

```diff
@@ -18,7 +18,7 @@
           </BIMDataButton>
         </div>
       </div>
-      <div class="tags-main__content__list-add">
+      <div v-if="isProjectAdmin(project)" class="tags-main__content__list-add">
```

Comment:
I suggest to put that `v-if` on the "add tag" button directly to avoid hiding the list title and preserve UI consistency.